account_brand/models/account_invoice.py

```diff
@@ -16,3 +16,10 @@ class AccountInvoice(models.Model):
             'cancel': [('readonly', True)],
         }
     )
+
+    @api.constrains('brand_id', 'company_id.brand_use_level', 'type')
```

Comment:
The [odoo 13 documentation](https://github.com/odoo/odoo/blob/88a489ba3e9e9c75bb86638294abe375faefd8b4/odoo/api.py#L119-L121) still says that constrains does not support dotted field dependencies.

Reply:
Yes sorry. My fault :see_no_evil: